submit/templates/submit/file_upload.html

```diff
@@ -0,0 +1,166 @@
+{% extends "submit/base.html" %}
+
+{% macro display_tree(key, item, even) %}
+  {% if key %}
+  <li class="{% if item.errors %}is-warning{% elif item.modified and item|just_updated %}is-success{% endif %} {% if even %}even{% endif %}">
+  {% endif %}
+    {% if item.name %}
+    <div class="columns">
+      <div class="column">
+        {{ item.name }}
+      </div>
+      <div class="column is-one-third-tablet">
+          <div class="columns is-mobile">
+            <div class="column">{{ item.size|filesizeformat }}</div>
+            <div class="column">{{ item.modified|timesince }}</div>
+            <div class="column has-text-centered is-narrow">
+             <a href="{{ url_for('ui.file_delete', submission_id=submission_id) }}?path={{ item.path }}">
+               <span class="icon">
+                 <i class="fa fa-trash"><span class="sr-only">Delete this file</span></i>
+               </span>
+             </a>
+            </div>
+          </div>
+      </div>
+    </div>
+    {% else %}
+    {% if key %}<span class="icon"><i class="fa fa-folder-open-o"></i></span>{{ key }}/{% endif %}
+    <ol class="{% if not key %}file-tree{% endif %}" style="list-style-type: none;">
+      {% for k, subitem in item.items() %}
```

Comment:
![Screenshot 2019-06-04 16 17 10](https://user-images.githubusercontent.com/746253/58911017-ee49f300-86e4-11e9-8268-2a58ef3bec82.png)

Nit: python's lexicographic sorting feels unnatural for a list of files/folders since it's not how most operating systems (AFAIK) sort files/folder names. I would be inclined to apply `string.casefold()` before sorting.

Reply:
Created https://github.com/arXiv/arxiv-submission-ui/issues/83